content/docs/Configuration/examples.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Examples 
+date: 2022-09-22
+weight: 1
+---
+
+# Configuration examples 
+
+## Examples for actions
+You can find detailed documentation for actions [here](/docs/actions).
+
+### Getting version
+{{< details title="Getting version from specfile" >}}
+```yaml
+    get-current-version:
+    - grep -oP '^Version:\s+\K\S+' my-package.spec
+```
+{{< /details >}}
+
+{{< details title="Getting version for Python packages with python3" >}}
+```yaml
+    get-current-version:
+    - python3 setup.py --version
+```
+{{< /details >}}
+
+### Manipulating spec file
+{{< details title="Downloading specfile from the dist-git repo" >}}
+```yaml
+  post-upstream-clone:
+- "wget https://src.fedoraproject.org/rpms/my-package/raw/main/f/my-package.spec -O my-package.spec"
+```
+{{< /details >}}
+
+{{< details title="Setting the Sources correctly in case of multiple Sources" >}}
+```yaml
+  fix-spec-file:
+  # define one of the Source variables correctly
+  - sed -i my_specfile_path -e "s/https.*only-vendor.tar.xz/my_correct_tarball_path/"
+  # fill in %release as if packit would have done it
+  - bash -c "sed -i my_specfile_path -r \"s/Release:(\s*)\S+/Release:\1${PACKIT_RPMSPEC_RELEASE}%{?dist}/\""
+```
+{{< /details >}}
+
+### Custom archive creation
+{{< details title="Creating archive with custom make target" >}}
+```yaml
+  create-archive:
+    - make release
+    - bash -c "ls -1t ./my-package-*.tar.gz | head -n 1"
+```
+{{< /details >}}
+
+{{< details title="Creating archive for Python packages with python3" >}}
+```yaml
+  create-archive:
+    - python3 setup.py sdist --dist-dir .
+    - bash -c "ls -1t ./my-package-*.tar.gz | head -n 1"
+```
+{{< /details >}}
+
+### Custom changelog generation
+{{< details title="Using changelog entry from a file" >}}
+```yaml
+  changelog-entry:
+  - cat .changelog_entry
+```
+{{< /details >}}
```

Comment:
@mfocko You are using a very nice example of how to do this interactively, what about putting it here?..;)